GVFS/GVFS.Service/GVFS.Service.Mac.csproj

```diff
@@ -19,8 +19,17 @@
   <ItemGroup>
     <Compile Include="..\GVFS.PlatformLoader\PlatformLoader.Mac.cs" Link="Mac\PlatformLoader.Mac.cs" />
     <Compile Include="Configuration.cs" />
+    <Compile Include="Handlers\RequestHandler.cs" />

```

Comment:
Why do all these source files need to be listed?  I thought .NET Core projects included all of the source files by default.

Reply:
We specify this setting in `GVFS.Service.Mac.csproj`

```
<EnableDefaultCompileItems>false</EnableDefaultCompileItems>
```

So that Windows specific files in the GVFS.Service directory don't get automatically included in the Mac project.